### DISCUSSION TITLE
The bucket of metrics is lost

### DISCUSSION THREAD

**Discussion by muyuhui** [open-telemetry/opentelemetry-java-instrumentation#11345] [Q&A]
I'm using opentelemetry-javaagent.jar. When I used version 1.26.0, there was `http_server_requests_milliseconds_bucket` in metrics, but when I upgraded to 1.27.0, `http_server_requests_milliseconds_bucket` was missing. Are there any configuration changes between these two versions that will affect metrics?

**Comment by laurit** [comment]
Probably https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/8334

**Reply by muyuhui** [reply]
I have added the following configuration, but I still don't see `http_server_requests_milliseconds_bucket`. Is my configuration wrong?
```
  otel.exporter.otlp.endpoint: http://otel-collector.tracing.svc.cluster.local:4317
  otel.exporter.otlp.protocol: grpc
  otel.propagators: b3multi

  # New
  otel.instrumentation.micrometer.prometheus-mode.enabled: 'true'
  otel.instrumentation.micrometer.histogram-gauges.enabled: 'true'
  otel.instrumentation.micrometer.base-time-unit: milliseconds
```
